actor/src/main/scala/org/apache/pekko/actor/ActorSystem.scala

```diff
@@ -23,6 +23,7 @@ import scala.collection.immutable
 import scala.concurrent.{ Await, ExecutionContext, ExecutionContextExecutor, Future, Promise }
 import scala.concurrent.blocking
 import scala.concurrent.duration.Duration
+import scala.jdk.DurationConverters._
```

Comment:
* we need to use pekko util because of Scala 2.12 res in 1.x
* is this a big win for 1.x?

Reply:
> is this a big win for 1.x?

I would like to make sure that currently maintained branches have the same behaviour/code